### PR TITLE
Test

### PR DESCRIPTION
**Pull request by VSemenovykh**
Test

### CODE REVIEW

File: autoads-api/pom.xml [hunk 1]
```diff
@@ -10,28 +10,27 @@
...
+<!--            <groupId>org.junit.jupiter</groupId>-->
+<!--            <artifactId>junit-jupiter-api</artifactId>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
+<!--        <dependency>-->
+<!--            <groupId>junit</groupId>-->
+<!--            <artifactId>junit</artifactId>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
+    </dependencies>
```
Comment (AndrewSmrnv) [inline autoads-api/pom.xml:34]: Delete redundant comment

File: autoads-impl/src/main/java/ru/ncedu/implement/ValidDataAutoAdsImpl.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            if (!(bodyStyles[i].name().equals(dataAuto.getBodyStyleType()))) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    @Override
+    public boolean EmailValidator(String email) {
+        final String EMAIL_PATTERN = "^[a-z0-9._%+-]+@[a-z0-9.-]+\\.[a-z]{2,4}$";
```
Comment (AndrewSmrnv) [inline autoads-impl/src/main/java/ru/ncedu/implement/ValidDataAutoAdsImpl.java:137]: Please extract this pattern as class constant

File: autoads-impl/src/main/java/ru/ncedu/implement/ValidDataAutoAdsImpl.java [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+        pattern = Pattern.compile(EMAIL_PATTERN);
+        matcher = pattern.matcher(email);
+
+        return matcher.matches();
+    }
+
+    @Override
+    public boolean PhoneValidator(String phone) {
+        final String PHONE_PATTERN = "^(\\+7)\\([0-9]{3}\\)+\\-[0-9]{3}\\-[0-9]{2}-[0-9]{2}$";
```
Comment (AndrewSmrnv) [inline autoads-impl/src/main/java/ru/ncedu/implement/ValidDataAutoAdsImpl.java:150]: Please extract this pattern as class constant

File: autoads-impl/src/test/resources/image_auto.sql [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+insert into image_auto (id, image_name, raster) values (3,'A8', '');
+insert into image_auto (id, image_name, raster) values (4,'FIESTA', '');
+insert into image_auto (id, image_name, raster) values (5,'FOCUS', '');
+
+/*global, LOAD_FILE, local */
+/*insert into image_auto (id, image_name, raster) values (1,'A3', local('./image-auto/Audi/AUDI-A3.JPG'));
+insert into image_auto (id, image_name, raster) values (2,'A4', local ('./image-auto/Audi/AUDI-A4.JPG'));  
+insert into image_auto (id, image_name, raster) values (3,'A8', local('./image-auto/Audi/AUDI-A8.JPG'));  
+insert into image_auto (id, image_name, raster) values (4,'FIESTA', local('./image-auto/Ford/FORD-FIESTA.JPG')); 
+insert into image_auto (id, image_name, raster) values (5,'FOCUS', local('./image-auto/Ford/FORD-FOCUS.JPG'));*/
```
Comment (AndrewSmrnv) [inline autoads-impl/src/test/resources/image_auto.sql:12]: Remove the comment

File: autoads-api/pom.xml (hunk 1 above)
Comment (VSemenovykh) [reply autoads-api/pom.xml:22]: done

File: autoads-impl/src/main/java/ru/ncedu/implement/ValidDataAutoAdsImpl.java (hunk 3 above)
Comment (VSemenovykh) [reply autoads-impl/src/main/java/ru/ncedu/implement/ValidDataAutoAdsImpl.java]: done

File: autoads-impl/src/test/resources/image_auto.sql (hunk 4 above)
Comment (VSemenovykh) [reply autoads-impl/src/test/resources/image_auto.sql]: done

**Pull request merged**